src/core/Select/Select.tsx

```diff
@@ -196,6 +197,8 @@ export const Select = <T,>(props: SelectProps<T>): JSX.Element => {
   const toggle = () => setIsOpen((open) => !open);
 
   const selectRef = React.useRef<HTMLDivElement>(null);
+  const toggleButtonRef = React.useRef<HTMLSpanElement>(null);
```

Comment:
Seems you are not doing anything with `toggleButtonRef`

Reply:
Removed this ref